code/API_definitions/qod-api.yaml

```diff
@@ -341,17 +698,213 @@ components:
+        maxBandwidth:
+          $ref: "#/components/schemas/Bandwidth"
+          description: |
+            The maximim best effort bandwidth
+            Equal to minBandwidth when profile is guaranteed. 
+        minDuration:
+          $ref: "#/components/schemas/Duration"
+        maxDuration:
+          $ref: "#/components/schemas/Duration"
+        priority:
+          type: float
+          example: 5.0
+          description: |
+            Priority for this session
+          format: float
+          minimum: 0
+          maximum: 10
+          default: 5
+        packetDelayBudget:
+          $ref: "#/components/schemas/Duration"
+        packetErrorLossRate:
+          type: int
+          description: |
+            The exponential power of the allowable error loss rate.
+            For instance 10 would be an error loss rate of 2 to the power of -10 (0.0009765625)
+          format: int32
+          minimum: 1
+          maximum: 10
+          example: 5
+          default: 1
+    CreateQosProfile:
+      description: Form to create a QosProfile
+      type: object
+      properties:
+        name:
+          type: string
+          example: foo-bar
+          description: Descriptive name for QoD Profile
+          minLength: 3
+          maxLength: 128
+          format: string
+          required: true
+        status:
+          $ref: "#/components/schemas/QosProfileStatusEnum"
+          required: true
+        minBandwidth:
+          $ref: "#/components/schemas/Bandwidth"
+          description: The minmum guaranteed bandwidth.  Set to zero when no guarontted bandwidth.
+          required: true
+        maxBandwidth:
+          $ref: "#/components/schemas/Bandwidth"
+          description: |
+            The maximim best effort bandwidth
+            Equal to minBandwidth when profile is guaranteed. 
+          required: true
+        minDuration:
+          $ref: "#/components/schemas/Duration"
+        maxDuration:
+          $ref: "#/components/schemas/Duration"
+        priority:
+          type: float
+          example: 5.0
+          description: |
+            Priority for this session
+          format: float
+          minimum: 0
+          maximum: 10
+          default: 5
+        packetDelayBudget:
+          $ref: "#/components/schemas/Duration"
+        packetErrorLossRate:
+          type: int
+          description: |
+            The exponential power of the allowable error loss rate.
+            For instance 10 would be an error loss rate of 2 to the power of -10 (0.0009765625)
+          format: int32
+          minimum: 1
+          maximum: 10
+          example: 5
+          default: 1
+    UpdateQosProfile:
+      description: Data type with attributes of a QosProfile
+      type: object
+      properties:
+        name:
+          type: string
+          example: foo-bar
+          description: Descriptive name for QoD Profile
+          minLength: 3
+          maxLength: 64
```

Comment:
The create is 128, but this update is 64. Should make them match. I think 128 might be best.